src/alda/AldaClient.java

```diff
@@ -74,7 +74,8 @@ public static void updateAlda()
 
     if (downloadURL == null) {
       throw new UnsuccessfulException(
-        "Alda download link not found for your platform."
+        "Alda download link not found for your platform. Please file an " +
+        "issue at: https://github.com/alda-lang/alda/issues/new"
```

Comment:
Yeah, I had to think about this for a minute. I think this is the one and only situation where it might be appropriate to file an issue at alda-lang/alda, since the issue is concerning a release. This does feel a little weird when the only open issue is an issue that blatantly says "don't file issues here" 😅 

Another option would be to file an issue in this repo (alda-lang/alda-client-java), since technically, the client did fail to find the download. What do you think of that idea?

Good catch about alda-repl-clj -- fixed.